activerecord/test/cases/serialized_attribute_test.rb

```diff
@@ -403,6 +403,27 @@ def test_values_cast_from_nil_are_persisted_as_nil
     assert_equal [topic, topic2], Topic.where(content: nil).sort_by(&:id)
   end
 
+  def test_serialized_attribute_with_default_can_update_to_default
+    # MySQL doesn't support default values for text columns, so we need to skip this test for MySQL
+    skip if ActiveRecord::Base.connection.adapter_name =~ /mysql/i
```

Comment:
We have a helper for this, see `current_adapter?(:Mysql2Adapter)`.

Also generally we put it around the test instead of skipping.

Reply:
Awesome! thank you! I didn't know about that
Done